polyaxon/values.yaml

```diff
@@ -490,6 +490,10 @@ rabbitmq:
             topologyKey: "kubernetes.io/hostname"
 
 docker-registry:
```

Comment:
It would be nice to have a similar comment as in [line 419](https://github.com/polyaxon/polyaxon-chart/pull/23/files#diff-43bc6296fbc9a23e12a4cce5c0f46e0eL419)

Reply:
Done 👍